scripts/asm-installer/install_asm

```diff
@@ -166,16 +166,18 @@ OPTIONS:
   -p|--project_id        <ID>         The GCP project ID
   -m|--mode              <MODE>       The type of installation to perform.
                                       Allowed values for <MODE> are
-                                      {install|migrate}. Choose --mode=install
-                                      for a new ASM installation or
-                                      --mode=migrate for an existing OSS
-                                      installation to be migrated to ASM.
-                                      MODE=install defaults to using Mesh CA
-                                      as the certificate authority.
-  -c|--ca                <CA>         The type of certificate authority to be
-                                      used. Allowed values for <CA> are
-                                      {citadel|mesh_ca}.
-
+                                      {install|migrate}. Choose MODE=install
+                                      for a new ASM installation or MODE=migrate
+                                      for an existing OSS installation to be
+                                      migrated to ASM. MODE=install defaults to
+                                      using Mesh CA as the certificate
+                                      authority.
+
+  -c|--ca                <CA>         (optional) The type of certificate
```

Comment:
If it's required for migrate mode, let's not put the (optional) tag in front.

Reply:
I am following the precedent from

 -k|--key_file          <FILE PATH>  (optional) The key file for a service
                                      account. Required if --service_account is
                                      passed.

Reply:
That's a good point. When I wrote it, service account and key file are both optional, so it made sense. This one seems different to me because the dependency is on a mandatory option, not an optional one.

If this might be confusing for users, I'd honestly rather just remove the (optional) from the beginning altogether and let users read the descriptions for more info. I don't see a way to have the two cases be consistent to everyone otherwise.

I won't block on this if you think this is the right way to handle it.